index.js

```diff
@@ -110,6 +110,11 @@ const runChecks = async (context, checkEvent) => {
               checkPullRequestLabelsModule.checkForIssueLabel(context)

```

Comment:
Since this feature listens for a new event, there should be an entry where oppiabot listens for the event. Similar to https://github.com/oppia/oppiabot/blob/ec404f432cf9553fbc184cc51bd572bf7e6fc806/index.js#L237-L241

Reply:
Done